docs/basics/sso/setting-up-sso.md

```diff
@@ -20,14 +20,22 @@ Sauce Labs supports Identity Provider (IdP)-initiated and Service Provider (SP)-
 First, set up a new SAML application in your [Identity Provider (IdP)](#setting-up-identity-provider). Next, enable and configure SSO in [Sauce Labs Team Management Panel](#integrating-with-sauce-labs-service-provider).
 
 :::note
-SSO at Sauce Labs is global. In other words, it does not matter in which region 
+SSO at Sauce Labs is global. In other words, it does not matter in which region
 you set it up. Unlike to the [old SSO implementation](/basics/sso-deprecated/setting-up-single-sign-on), which has been deprecated.
 :::
 
 ## Setting Up Identity Provider
 
+:::tip
+We provide ready-made SAML applications for some identity providers. They allow you to set up SSO integration with just a few clicks. Choose your provider and follow the configuration tutorial:
```

Comment:
Nitpick: I would change `ready-made` to `preconfigured`

Reply:
fixed